packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -19,6 +19,7 @@ const Translate = {
   'eu-south-1': 'eu-central-1',
   'ap-northeast-1': 'ap-northeast-1',
   'ap-northeast-2': 'ap-northeast-2',
+  'ap-northeast-3': 'ap-northeast-1',
```

Comment:
Translate is not available in ap-northeast-3. Falling back to ap-northeast-3

https://docs.aws.amazon.com/general/latest/gr/translate-service.html